doc/source/user_guide/tui_commands.rst

```diff
@@ -22,79 +22,73 @@ you can use the Python built-in `help <https://docs.python.org/3/library/functio
 and `dir <https://docs.python.org/3/library/functions.html#dir>`_ functions.
 
 The arguments to a TUI command are those that would be passed in direct
-interaction in the Fluent console, but they are in a Pythonic style. The most
-productive way to write Python commands is with reference to existing TUI
-commands. The following examples show how the Python usage mirrors the existing
-TUI usage.
+interaction in the Fluent console, but they are in a Pythonic style. In the recent
+Fluent versions, in both meshing and solution mode,
+you can use Python journaling, which is a beta feature,
+to construct the TUI commands for PyFluent. The following section describes how to
```

Comment:
Should we mention somewhere how to perform Python journaling?

As far as I've seen, we don't have that info anywhere in the PyFluent documentation. The current PyFluent journaling documentation seems to be mostly useless/empty: 
https://fluent.docs.pyansys.com/version/stable/api/general/index.html#journaling

Reply:
This is part of Fluent documentation.